pgmpy/inference/base.py

```diff
@@ -80,3 +81,25 @@ def __init__(self, model):
             for factor in model.get_factors():
                 for var in factor.variables:
                     self.factors[var].append(factor)
+
+        elif isinstance(model, DynamicBayesianNetwork):
+            # This method constructs a temporary Bayesian Model so as to create the
+            # initial potential for inference.
+            # This will also declare the following two parameters
+            # 1) interface nodes:- These are the nodes whose children are in the 
+            # first timeslice.
+            # 2) one_and_half_model:- This is the bayesian model which consists of the
+            # interface nodes along with the first time slice nodes.
+            # Both of the above parameters will be required by the junction tree algorithm
+            # too.
+            start_model = BayesianModel(model.edges())
```

Comment:
Couldn't get sense of lines 95 - 97. Replacing them with `self.start_bayesian_model = model` would do the same thing ??